Character/Classes/Artificer/Artificer.xml

```diff
@@ -143,7 +143,7 @@
 				<text/>
 				<text>Spellcasting Focus</text>
 				<text/>
-				<text>You can use an arcane focus as a spellcasting focus for your artificer spells. See chapter 5, “Equipment,” in the Player's Handbook for various arcane focus options.</text>
+				<text>You can use an arcane focus as a spellcasting focus for your artificer spells. See chapter 5, “Equipment”, in the Player's Handbook for various arcane focus options.</text>
```

Comment:
And unicode open and closing brackets - does this want to be made consistent and back to simple ascii?

Reply:
(or vica-versa)

Reply:
I think I recent saw some commits by @Cphrampus converting them to simple quotes so that is probably the way to go.

Reply:
I hadn't noticed them, but it looks significantly better this way ;-)